src/Model/Entity/Traits/DataTransferObjectTrait.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Friendica\Model\Entity\Traits;
+
+use Friendica\Network\HTTPException;
+
+trait DataTransferObjectTrait
+{
+	public function __set($name, $value)
```

Comment:
This one catches situations, where we want to override Data Transfer Objects (= Api, Mastodon Instance, ...)